packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -1116,6 +1116,8 @@ function detachFiber(current: Fiber) {
   current.memoizedProps = null;
   current.stateNode = null;
   if (alternate !== null) {
+    // Stop the alternate from going back to the current fiber.
+    alternate.alternate = null;
```

Comment:
This doesn't do anything either, right? We're passing it to `detachFiber` which does the same thing.

Reply:
It does the same thing, but it avoids an extra function call. So in the scheme of things, it's slightly better.

Reply:
We should still probably get this change in though :)
https://github.com/facebook/react/pull/18521#discussion_r404907057

Reply:
Ah, it looks like you've included it on another PR.